src/pages/catalog/oer/index.tsx

```diff
@@ -0,0 +1,244 @@
+        <div className="container-md">
+          <div className={styles.grid}>
+            <SharedQuestionCard
+              image={eorphysics1}
+              title="Physics: Mechanics"
+              href="https://us.prairielearn.com/pl/public/course/2923/questions"
+              github="https://github.com/PrairieLearn/pl-oer-physics1"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eorphysics2}
+              title="Physics: E&amp;M"
+              href="https://us.prairielearn.com/pl/public/course/2922/questions"
+              github="https://github.com/PrairieLearn/pl-oer-physics2"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eorstatics}
+              title="Statics"
+              href="https://us.prairielearn.com/pl/public/course/2924/questions"
+              github="https://github.com/PrairieLearn/pl-oer-statics"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eordynamics}
+              title="Dynamics"
+              href="https://us.prairielearn.com/pl/public/course/2955/questions"
+              github="https://github.com/PrairieLearn/pl-oer-dynamics"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eorsolids}
+              title="Solid Mechanics"
+              href="https://us.prairielearn.com/pl/public/course/2925/questions"
+              github="https://github.com/PrairieLearn/pl-oer-solidMechanics"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eornumerical}
+              title="Numerical Methods"
+              href="https://us.prairielearn.com/pl/public/course/2926/questions"
+              github="https://github.com/PrairieLearn/pl-oer-numericalMethods"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eorthermo}
+              title="Thermodynamics"
+              href="https://us.prairielearn.com/pl/public/course/2988/questions"
+              ownerName="Shelby Hutchens"
+              ownerEmail="hutchs@illinois.edu"
+              github="https://github.com/PrairieLearn/pl-oer-thermodynamics"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={eorpython}
+              title="Introduction to Python"
+              href="https://us.prairielearn.com/pl/public/course/2889/questions"
+              github="https://github.com/PrairieLearn/pl-oer-python"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={cs233Image}
+              title="Miscellaneous CS Topics"
+              href="https://us.prairielearn.com/pl/public/course/6717/questions"
+              github="https://github.com/PrairieLearn/pl-oer-csmix"
+            ></SharedQuestionCard>
+
+            <SharedQuestionCard
+              image={sigcse2023}
+              title="SIGCSE 2023 - Workhop"
```

Comment:
SIGCSE 2024?

Reply:
actually, because of the image, I think we should keep as SIGCSE 2023. We can later decide what we will do with with the course instance used for the affiliated section during SIGCSE 2024. Once Firas creates the course instance, I will add a link to the current affiliated section page.

Reply:
Ah, ok, that makes sense. I just had SIGCSE 2024 on the brain!